docs/source/user_guide/pipelines.md

```diff
@@ -359,19 +393,25 @@ Before you can export a pipeline on Kubeflow Pipelines or Apache Airflow you mus
 #### Exporting a pipeline from the Visual Pipeline Editor
 
 To export a pipeline from the Visual Pipeline Editor:
-1. Click `Export Pipeline` in the editor's tool bar.
+1. Click `Export Pipeline` in the editor's toolbar.
 
    ![Open pipeline run wizard](../images/user_guide/pipelines/pipeline-editor-export.png)
 
-1. For generic pipelines select a runtime platform (local, Kubeflow Pipelines, or Apache Airflow) and a runtime configuration for that platform. For runtime-specific pipelines select a runtime configuration.
+2. For generic pipelines select a runtime platform (local, Kubeflow Pipelines, or Apache Airflow) and a runtime configuration for that platform. For runtime-specific pipelines select a runtime configuration.
 
-1. Select an export format.
+3. Select an export format.
 
-1. Customize your file name using the Export Filename box
+4. Customize your file name using the Export Filename box
    
    ![Configure pipeline export options](../images/user_guide/pipelines/configure-pipeline-export-options.png)
 
-1. Import the exported pipeline file using the Kubeflow Central Dashboard or add it to the Git repository that Apache Airflow is monitoring.
+5. Configure pipeline parameters, if applicable. If any nodes reference parameters defined in the `Pipeline Parameters` panel, the value these parameters take can be customized here. If a parameter is marked as required and no default value is set, a value must be provided before the `OK` button is enabled.
+
+   ![Configure pipeline export options with parameters](../images/user_guide/pipelines/configure-pipeline-export-options-parameters.gif)
```

Comment:
The third parameter name is `should_exit`. Without context it's hard to say whether you meant it to be that, or `should exist`.  Did you?

Reply:
yeah I did mean `should_exist` ha. I'll probably just rename is completely and re-record them 😄